### PR TITLE
Fix Issue 19064 - [REG2.081] Vibe.d's InterfaceProxy no longer works

### PR DESCRIPTION
**Pull request by RazvanN7**

### CODE REVIEW

**Comment by RazvanN7** [comment]
Trying to rebase onto stable leads to a lot of conflicts :-?

**Comment by jacob-carlborg** [comment]
Delete the branch, create a new branch with the same name but based on stable, cherrypick the commit.

**Comment by wilzbach** [comment]
Yep, listen to what dlang-bot tells you ;-)
(rebased for you)

File: test/compilable/test17373.d [hunk 1]
```diff
@@ -17,4 +18,15 @@ interface Simple
...
+interface OutputStream{}
+interface Stream : InputStream, OutputStream{}
+interface ConnectionStream : Stream
+{
+    @property bool connected() const;
+    void close();
+}
+
+static assert(__traits(getOverloads, ConnectionStream, "connected").length == 1);
+static assert(__traits(getOverloads, ConnectionStream, "close").length == 1);
```
Comment (wilzbach) [inline test/compilable/test17373.d]: Why not `.stringof` and do a full comparison?

Comment (RazvanN7) [reply test/compilable/test17373.d]: It's the same thing.

Comment (wilzbach) [reply test/compilable/test17373.d]: Yes, but it ensures that you don't accidentally return the wrong one. Let's better be safe than sorry.
I added this as myself and hope you don't mind?

**Review by wilzbach — APPROVED** [review]
Thanks!

**Comment by wilzbach** [comment]
> Trying to rebase onto stable leads to a lot of conflicts.

Maybe your `upstream` wasn't up to date? Try `git fetch upstream` if you run into this the next time.

**Pull request merged**